docs/developer/base/basics.md

```diff
@@ -1,3 +1,165 @@
 # Basics
 
 -----
+
+The AgentCheck base class in base.py contains the logic that all Checks inherit.
+
+In addition to the integrations inheriting from AgentCheck, other classes that inherit from AgentCheck include:
+
+- [PDHBaseCheck](https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/datadog_checks/base/checks/win/winpdh_base.py)
+- [OpenMetricsBaseCheck](https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/datadog_checks/base/checks/openmetrics/base_check.py)
+- [GenericPrometheusCheck](https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/datadog_checks/base/checks/prometheus/base_check.py)
+- [PrometheusCheck](https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/datadog_checks/base/checks/prometheus/prometheus_base.py)
+- [KubeLeaderElectionBaseCheck](https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/datadog_checks/base/checks/kube_leader/base_check.py)
+
+
+## Getting Started
+In each Check, Datadog looks for `__version__` and a subclass of AgentCheck at the root of every package.
+
+Below is an example of the `__init__.py` folder for AwesomeCheck:
+```python
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from .__about__ import __version__
+from .check import AwesomeCheck
+
+__all__ = ['__version__', 'AwesomeCheck']
+```
+
+Note that we look for `__version__` in `__about__.py` in this case, however it does not necessarily need to be put there. 
```

Comment:
```suggestion
Note that Datadog looks for `__version__` in `__about__.py`. However, it does not necessarily need to be put there. 
```

Could you also just explain this a little more? Does it default to anything if it isn't there?

Reply:
Adding default value